### PR TITLE
Increase the limited log file size

### PR DESCRIPTION
**Pull request by yanhongwangg**
Follow the previous configuration，Too many files will be generated。
image

### CODE REVIEW

**Review by mychaow — APPROVED** [review]
LGTM

File: server/src/assembly/resources/conf/logback.xml [hunk 1]
```diff
@@ -65,7 +65,7 @@
         <file>${IOTDB_HOME}/logs/log_info.log</file>
         <rollingPolicy class="ch.qos.logback.core.rolling.SizeAndTimeBasedRollingPolicy">
             <fileNamePattern>${IOTDB_HOME}/logs/log-info-%d{yyyyMMdd-HH}.%i.log.gz</fileNamePattern>
-            <maxFileSize>50MB</maxFileSize>
+            <maxFileSize>1GB</maxFileSize>
```
Comment (HTHou) [inline server/src/assembly/resources/conf/logback.xml:68]: Is 1GB too large?

File: server/src/assembly/resources/conf/logback.xml [hunk 2]
```diff
@@ -46,7 +46,7 @@
         <file>${IOTDB_HOME}/logs/log_warn.log</file>
         <rollingPolicy class="ch.qos.logback.core.rolling.SizeAndTimeBasedRollingPolicy">
             <fileNamePattern>${IOTDB_HOME}/logs/log-warn-%d{yyyyMMdd-HH}.%i.log.gz</fileNamePattern>
-            <maxFileSize>10MB</maxFileSize>
+            <maxFileSize>200MB</maxFileSize>
```
Comment (wangchao316) [inline server/src/assembly/resources/conf/logback.xml:49]: If the value of maxfilesize is large and logs are queried and written during log compression, performance jitters may occur. This has been a problem in our production environment.

**Review by wangchao316 — CHANGES_REQUESTED** [review]

File: server/src/assembly/resources/conf/logback.xml (hunk 2 above)
Comment (yanhongwangg) [reply server/src/assembly/resources/conf/logback.xml:49]: emmm,What size do you think is appropriate

File: server/src/assembly/resources/conf/logback.xml (hunk 1 above)
Comment (yanhongwangg) [reply server/src/assembly/resources/conf/logback.xml:68]: emmm,maybe

File: server/src/assembly/resources/conf/logback.xml (hunk 2 above)
Comment (neuyilan) [reply server/src/assembly/resources/conf/logback.xml:49]: Hi, @wangchao316 do you have any suggestions according to your experience?

Comment (wangchao316) [reply server/src/assembly/resources/conf/logback.xml:49]: hi @neuyilan 
The log printing level and log printing frequency are distinguished.
Generally, the value of info is 35 MB.
Generally, the value of warn is 50 MB.
Generally, the debug value is 125 MB.

Comment (neuyilan) [reply server/src/assembly/resources/conf/logback.xml:49]: Hi, @wangchao316 Thanks a lot.
Hi @yanhongwangg, maybe you could revise your codes according to @wangchao316's suggestions.

**Pull request closed without merging**